### PR TITLE
Openapi validator 1st level

### PR DESCRIPTION
**Pull request by CodeSandwich**
1st level of OpenAPI validator, very basic, only checks if route is present in OpenAPI spec. Part of https://github.com/input-output-hk/jormungandr/issues/548

### CODE REVIEW

File: jcli/src/jcli_app/utils/open_api_verifier.rs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+use std::io;
+
+pub struct OpenApiVerifier(OpenApiVerifierMode);
+
+enum OpenApiVerifierMode {
+    AcceptAll,
+    Verify(OpenApiVerifierImpl),
+}
+
+struct OpenApiVerifierImpl {
```
Comment (mzabaluev) [inline jcli/src/jcli_app/utils/open_api_verifier.rs]: Let's not use "impl" in type names. `VerifierState`?

Comment (CodeSandwich) [reply jcli/src/jcli_app/utils/open_api_verifier.rs]: It's not quite state, more like implementation of the mode. I think I quite often see `...Inner` in Rust ecosystem, would `OpenApiVerifierInner` be ok? Why not `Impl`? Too much like if it was implementor of an `OpenApiVerifier` interface?

Comment (NicolasDP) [reply jcli/src/jcli_app/utils/open_api_verifier.rs]: I agree with @mzabaluev

Comment (CodeSandwich) [reply jcli/src/jcli_app/utils/open_api_verifier.rs]: I've renamed it to `pub OpenApiVerifier` -> `VerifierMode` -> `Verifier`, it's less verbose and names without `OpenApi` prefix are private anyway, so their OpenApi context is always clear

**Pull request merged**